src/assets/styles/outdated_browser.scss

```diff
@@ -0,0 +1,25 @@
+.outdated-browser {
+  display: none;
+  position: fixed;
+  top: 0;
+  bottom: 0;
+  left: 0;
+  right: 0;
+  z-index: 1000;
+  background: $dark;
+  color: $light-blue;
+  padding: 4em;
+  font-size: 1.3em;
+  text-align: center;
+  a {
+    &:link, &:visited, &:hover {
+      color: $aclu-blue;
+    }
+  }
+}
+
+.no-flexbox, .no-webgl {
+  .outdated-browser {
+    display: block;
+  }
```

Comment:
I get a weird flickering gray line on my comp (prob due to z-fighting between this overlay and mapbox). Simple solution was to just set display: none on the ```.event-map``` here as well. FYI, I got this after just forcing display:block on my mb-pro in chrome, might not be an actually issue on truly outdated browsers that can't load webgl or flexbox at all. However, traveling and don't have an easy ability to QA that at the moment.
<img width="829" alt="screen shot 2017-04-26 at 7 14 02 pm" src="https://cloud.githubusercontent.com/assets/4334343/25461075/e297f180-2ab4-11e7-8183-e5100272511c.png">